telescope/metrics_math.py

```diff
@@ -22,16 +22,15 @@
 def calculate_results_list(metric, input_datarows):
   datarows_to_return = []
 
-  if metric == "hop_count":
-    input_datarows = mlab.parse_pt_data(input_datarows)
-
   for datarow in input_datarows:
 
     calculated_result = None
 
     if metric == "hop_count":
       timestamp = datarow['log_time']
-      calculated_result = len(datarow['hops'])
+      calculated_result = datarow['paris_traceroute_hop_dest_ip'] + ', ' + \
+                          datarow['paris_traceroute_hop_src_ip'] + ', ' + \
+                          datarow['connection_spec_client_ip']
```

Comment:
I'm not clear on what we're doing here. This yields rows that look like this:

```
1391214842,"68.86.90.58, 68.86.83.105, 67.186.163.7"
```
I'm assuming you don't want to just pull out the number of hops in the route and you instead want to pull out the information needed to reconstruct the route. We should talk about the long term plan here because this is going to be a little trickier than the metrics were pulling already. We can construct a single row of download throughput, upload throughput, etc. metrics from a single row of BigQuery results, whereas with traceroute, we'll likely have to add some logic to convert a bunch of rows of PT into a single "path" row.